etl/importer.py

```diff
@@ -207,6 +208,10 @@ def import_psts(
     if clean:
         logger.warning(f"Deleting {account.title} account files (if exists)")
         account.files.all().delete()
+    if clean_file:
+        logger.warning(f"Deleting failed file for {account.title}")
+        # MVP: We assume there is only 1 failed file per account.
+        account.files.get(import_status=ratom.File.FAILED).delete()
```

Comment:
Might as well make this a ``filter()`` here. We want to avoid the importer failing for any reason before it can be caught in the PstImporter.